crawler.py

```diff
@@ -179,6 +179,10 @@ def run_all(driver):
 
 os.environ['PATH'] += os.pathsep + os.path.abspath('tools')
 os.environ['MOZ_HEADLESS'] = '1'
-driver = webdriver.Firefox(firefox_binary='tools/nightly/firefox-bin')
+
+# Webdriver uses Firefox Binaries from downloaded cov build
+my_path = os.path.abspath(os.path.dirname(__file__))
+filename = os.path.join(my_path, 'firefox/firefox-bin')
+driver = webdriver.Firefox(firefox_binary=filename)
```

Comment:
I think you can use the relative path here, no need to pass it an absolute path. Can you try if it works?

Reply:
I've tried it initially. It didn't work in that way; therefore, I've added an absolute path.

Reply:
This is strange, it works for me!

Reply:
Oops. Probably, it was before I have solved other problems. I can change it. :)

Reply:
@marco-c yep, it worked. :)